src/main/asciidoc/inc/_docker-copy.adoc

```diff
@@ -0,0 +1,33 @@
+[[docker:copy]]
```

Comment:
Without changes in other files, like "src/main/asciidoc/index.adoc", this file doesn't look being picked and included into the generated documentation.